src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py

```diff
@@ -78,6 +78,27 @@ def load_command_table(self, _):
         g.command('show', 'get_sql_container')
         g.command('delete', 'delete_sql_container')
 
+    with self.command_group('cosmosdb sql stored_procedure', cosmosdb_sql_sdk, client_factory=cf_sql_resources) as g:
+        g.custom_command('create', 'cli_cosmosdb_sql_stored_procedure_create_update')
+        g.custom_command('update', 'cli_cosmosdb_sql_stored_procedure_create_update')
+        g.command('list', 'list_sql_stored_procedures')
+        g.command('show', 'get_sql_stored_procedure')
+        g.command('delete', 'delete_sql_stored_procedure')
```

Comment:
Do you want to prompt confirmation message for delete command? It is recommended to add confirmation for delete command. e.g. https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/commands.py#L82